README.md

```diff
@@ -62,99 +108,80 @@ exports.process = function process(msg, cfg, snapshot) { };
 exports.shutdown = function shutdown(cfg, startupData) { };
 ```
 
-## Fixtures
-Fixtures are used in testing code as a location to store test instances within a codebase.
-
-For example, the ``process(msg, cfg)`` function takes at least 2 parameters:
+Furthermore, the provided fixture will be initially be checked against its respective `schema`, if the action/trigger has a static schema, for invalid message. This will not impede the running of the `process` action.
```

Comment:
```suggestion
Furthermore, if the action/trigger has a static schema, the provided fixture will be initially be checked against its respective `schema` for an invalid message. This will not impede the running of the `process` action.
```